action.yml

```diff
@@ -0,0 +1,31 @@
+name: Remote tflint Config
+description: Copies a tflint configuration file from a remote repo for local use
+author: ryanwholey
+inputs:
+  source-repo: 
+    required: true
+    description: The repository to copy a tflint configuration from
+  source-path:
+    description: The directory in the remote repo where the configuration exists
+    default: .
+  source-filename:
+    description: The source filename
+    default: .tflint.hcl
+  source-ref:
+    description: Ref or branch of the remote repository to target
+    default: main
+  destination-path:
```

Comment:
Similar to above, path means the whole path, I'd just merge this and filename

Reply:
Also, it would be nice to default to writing to `$HOME/.tflint.hcl`. Then TFLint would load the config by default, without the need to reference the step output. Seems like using `runner.temp` would really only be necessary/meaningful when using a self-hosted runner.

Reply:
Couldn't find a great way to include the home directory in the action.yml defaults, but i agree, using the home dir is a nice touch!